### ISSUE TITLE
Getting started docs

### ISSUE THREAD

**Issue by ryanmcnamara** [palantir/godel#166]
I don't think there are docs for getting started (getting a usable godelw added to a project)

Also docs on updating after the getting started would be nice too

**Comment by bmoylan** [comment]
Does this help? https://github.com/palantir/godel/wiki/Add-g%C3%B6del-to-a-project

**Comment by nmiyake** [comment]
Agreed -- in my mind, this is basically the last/main blocker for declaring 1.0 and pushing out more broadly.

The Wiki does cover some things, but I think there should be a lot more tutorials/guides around setting up specific workflows and enabling specific features.

**Comment by ryanmcnamara** [comment]
Nice, yea a link to brad's wiki page from the readme could even be sufficient
